src/xcode/ENA/ENA/Source/Services/HealthCertificateValidation/MockHealthCertificateValidationService.swift

```diff
@@ -16,12 +20,12 @@ struct MockHealthCertificateValidationService: HealthCertificateValidationProvid
 		].compactMap { $0 }
 	)
 
-	var validationResult: Result<HealthCertificateValidationReport, HealthCertificateValidationError> = .success(.validationPassed([]))
+	var validationResult: Result<HealthCertificateValidationReport, HealthCertificateValidationError> = .success(.validationFailed([validationResult1, validationResult2]))
 
 	func onboardedCountries(
 		completion: @escaping (Result<[Country], HealthCertificateValidationOnboardedCountriesError>) -> Void
 	) {
-		DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
+		DispatchQueue.main.asyncAfter(deadline: .now() + 0) {
```

Comment:
```suggestion
		DispatchQueue.main.asyncAfter(deadline: .now()) {
```

Reply:
This reminded me to clean up the mocks. I added this initially to check the loading state in the UI. The dispatch to the main queue should not be needed anymore, so I removed it and did some other cleanup.